package.json

```diff
@@ -48,7 +48,9 @@
     "compile-base-ui": "node ./.storybook/base-ui-compiler.js",
     "storybook:build": "build-storybook -s ./src/assets",
     "generate-stylable-index": "stc --srcDir=\"./dist/src/components\" --diagnostics --indexFile=index.st.css",
-    "generate": "wuf generate --codemods .wuf/generator/codemods --templates .wuf/generator/templates",
+    "generate": "node scripts/generate.js",
+    "generate:component": "wuf generate --codemods .wuf/generator/codemods --templates .wuf/generator/templates/desktop",
```

Comment:
both generator templates now have `src/components` folders. you could remove them and have just component folder with `--output` flag:

```
wuf generate --codemods .wuf/generator/codemods --templates .wuf/generator/templates/desktop --output src/components,
```

it's not critical at all, just thinking maybe templates shouldn't include `src/components` as part of them

Reply:
Excellent!